frontend/app/src/pages/locations/[identifier].vue

```diff
@@ -31,11 +31,8 @@ const location = computed<TradeLocationData>(() =>
     </v-row>
     <location-value-row class="mt-8" :identifier="identifier" />
     <location-assets class="mt-8" :identifier="identifier" />
-    <closed-trades :location-overview="identifier" class="mt-8" />
-    <deposits-withdrawals-content
-      :location-overview="identifier"
-      class="mt-8"
-    />
-    <ledger-action-content :location-overview="identifier" class="mt-8" />
+    <closed-trades :location-overview="identifier" />
+    <deposits-withdrawals-content :location-overview="identifier" />
+    <ledger-action-content :location-overview="identifier" />
```

Comment:
Is there any specific reason why the margin was ok before when we were hiding the tables and it is not ok now that they are always visible?

Reply:
I think it works before, except for the `ledger-action-content` that wrapped in fragment, we can't put the class directly here (we can wrap it in div).
But for the consistency I put all the margin directly inside the component now, so I can remove the margin here.